virtcontainers/qemu_arch_base.go

```diff
@@ -331,6 +338,8 @@ func (q *qemuArchBase) appendBridges(devices []govmmQemu.Device, bridges []Bridg
 			t = govmmQemu.PCIEBridge
 		}
 
+		b.Addr = bridgePCIStartAddr + idx
```

Comment:
Isn't this a duplicate of what is already in `qemu_amd64.go`? If so, can't this be removed?

Reply:
The code to add a bridge was different, and the address needs to be incremented while adding bridges in both the cases.